### PR TITLE
Update statistical.py

### PR DESCRIPTION
**Pull request by Paulyn01**

### CODE REVIEW

**Comment by Paulyn01** [comment]
its now failing all the test which I find confusing

**Review by simonetgordon — CHANGES_REQUESTED** [review]
Undo changes in `.idea/ivy.iml` and `.idea/misc.xml`, commit + push to branch in order to merge with master.

**Comment by Paulyn01** [comment]
how do I do that?

**Comment by simonetgordon** [comment]
You need to make sure that these files are exactly the same as in master and that changes don't appear in this PR. The simplest method might be to just copy and paste if the difference are small.

File: .idea/ivy.iml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
-    <content url="file://$MODULE_DIR$" />
-    <orderEntry type="jdk" jdkName="Remote Python 3.8.10 Docker (unifyai/ivy:latest)" jdkType="Python SDK" />
+   <content url="file://$MODULE_DIR$" />
+     <orderEntry type="jdk" jdkName="Remote Python 3.8.10 Docker (unifyai/ivy:latest)" jdkType="Python SDK" />
     <orderEntry type="sourceFolder" forTests="false" />
```
Comment (simonetgordon) [inline .idea/ivy.iml:6]: I think they're still appearing as different as the spacing is different.

**Review by simonetgordon — CHANGES_REQUESTED** [review]

File: .idea/ivy.iml (hunk 1 above)
Comment (Paulyn01) [reply .idea/ivy.iml:6]: alright.

**Review by simonetgordon — CHANGES_REQUESTED** [review]
The extra files are still appearing. There are just small things which need to be changed, like spacing and adding a blank line at the end of the file.

**Pull request merged**